src/citrine/_serialization/properties.py

```diff
@@ -498,10 +498,13 @@ def _deserialize(self, value: typing.Any) -> typing.Any:
                 return prop.deserialize(value)
             except ValueError:
                 pass
+            except RuntimeError:
+                # passes on RuntimeError from Property.deserialize_from_dict
+                pass
```

Comment:
I realize you are responding to existing code, but the error in `Property.deserialize_from_dict` should really be a ValueError.
[ValueError](https://docs.python.org/3.10/library/exceptions.html?highlight=runtimeerror#ValueError)
https://github.com/CitrineInformatics/citrine-python/blob/177795d6408d9667d053a25e2f26a3b4a079bbcd/src/citrine/_serialization/properties.py#L83-L103

Reply:
I'll have to fix a handful of tests that rely on that being a `RuntimeError`, but I can address it.